content/docs/buildpack-author-guide/create-buildpack/caching.md

```diff
@@ -143,11 +146,14 @@ with the new logic below that checks to see if any gems have been changed. This
 
 We'll now write additional metadata to our `bundler.toml` of the form `cache = true` and `launch = true`. This directs the lifecycle to cache our gems and provide them when launching our application. With `cache = true` the lifecycle can keep existing gems around so that build times are fast, even with minor `Gemfile.lock` changes.
 
+Note that there may be times when you would want to clean the cached layer from the previous build, in which case you should always ensure to remove the contents of the layer before proceeding with the build. In the case below this can be done using a simple `rm -rf "$bundlerlayer"/*` after the `mkdir -p "$bundlerlayer"` command.
```

Comment:
This tripped me up quite a bit when I first started writing a buildpack based off this tutorial. I didn't quite understand that caching would essentially restore the previous layer and we would have to manually clear the contents if we want to have a clean environment when something like our deps change.